src/main/java/net/openhft/chronicle/bytes/HexDumpBytes.java

```diff
@@ -1106,11 +1122,11 @@ private void doIndent() {
 
     @Override
     @NotNull
-    public Bytes<Void> write(@NotNull byte[] bytes, int offset, int length)
+    public Bytes<Void> write(@NotNull byte[] byteArray, int offset, int length)
```

Comment:
Why do we use @NotNull for byte arrays sometimes and not other times (even though we assert not null)

Reply:
Originally there was a misunderstanding about what `@NotNull` means for arrays. I raised a new issue about this:
https://github.com/OpenHFT/Chronicle-Bytes/issues/291